invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js

```diff
@@ -34,6 +34,15 @@ import {
 import { axiosWithconfig, SearchItemCreators } from "../../utils";
 import { DashboardResultView, DashboardSearchLayoutHOC } from "./base";
 
+const statuses = {
+  in_review: { color: "yellow", title: i18next.t("In review") },
```

Comment:
given that such colors are not used in a `grid`, are they matching the upload form?

Reply:
The colors roughly match the colors used in the upload form: https://github.com/inveniosoftware/react-invenio-deposit/blob/93d6a0d2ae057f4031b50e614aef33d4acb2a1ac/src/lib/components/DepositStatus/DepositStatusBox.js

Reply:
on another topic: those colors are not easily overridable or extendable in the current implementation, they are hardcoded